api/lib/source/polkadotV0-source.js

```diff
@@ -775,6 +775,16 @@ class polkadotAPI {
       })
     )
     const votesSum = proposal.seconds.length
+
+    // mimicing the spend period in polkadot UI (not sure this is correct)
```

Comment:
How can we confirm? There is no explorer displaying this information, is there?

Reply:
check proposals in motions and compare

Reply:
and for democracy?

Reply:
I checked polkadot.js UI

Reply:
I don't see this information anywhere in there :eyes: 

![image](https://user-images.githubusercontent.com/40721795/92751354-2cbe6b00-f388-11ea-96bc-b6e2c994368a.png)

Reply:
![image](https://user-images.githubusercontent.com/5869273/92768613-c7558480-f35d-11ea-9825-b966289518a7.png)